client/instrumenter/pom.xml

```diff
@@ -45,6 +43,10 @@
       <groupId>org.hawkular.apm</groupId>
       <artifactId>hawkular-apm-collector</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.hawkular.apm</groupId>
+      <artifactId>hawkular-apm-trace-publisher-rest-client</artifactId>
```

Comment:
Possibly this is dependent upon the outcome of the previous comment, as to whether it is necessary.